options.html

```diff
@@ -7,16 +7,25 @@
     <style type="text/css">
       body {
           font: message-box;
-          text-rendering: optimizeLegibility;
           display: flex;
           flex-direction: column;
           max-width: 650px;
           margin: 0 auto;
       }
+      table {
+          margin-bottom: 3px;
+          width: 100%;
+      }
+      td:nth-child(1) {
+          width: 16em;
+      }
+      td:nth-child(2) {
+          width: 0;
+          padding-right: 2px;
```

Comment:
```suggestion
```
This padding-right causes uneven spacing between the select and input elements on both sides, please remove.

Reply:
Interesting.  Which OS and browser did you use for testing?

Reply:
Linux, the current 95.0.1.